### ISSUE TITLE
{JC,JCEC}.raw.input.[AS].txt files' format

### ISSUE THREAD

**Issue by andrepessoni** [Xinglab/rmats-turbo#231]
Hello,

I am currently using rMATS to identify alternative splicing events between several samples (.bam) for two conditions, but I need to perform batch correction and normalization of the junction counts.
After reading Issue #111 , I'm working with JCEC.raw.input.[AS].txt files and I plan to correct the counts and use the new file to run the statistics, but I need the sample IDs to be able to do this correctly and they are not present in the raw file. Are the columns (inside IJC_SAMPLE_{1,2} and SJC_SAMPLE_{1,2}) in the same order as in the input list file (--b1 and --b2)?
I'm also struggling on how to identify the events (in the raw files we only have numeric IDs) if I want to run the statistics on my own, is there a way to match them with gene IDs present in the annotation?

Thank you very much!

**Comment by EricKutschera** [comment]
Yes the values in the output files are in the same order as --b1 and --b2. For example the values in the column IJC_SAMPLE_1 are in the same order as the bams given with --b1 and similarly IJC_SAMPLE_2 is in the same order as the bams from --b2

**Comment by andrepessoni** [comment]
Thank you for the quick reply!
Just to confirm one more thing, the IncFormLen and SkipFormLen values should not be corrected and must remain the same for adequate normalization, right?
Thanks again for all the help!

**Comment by EricKutschera** [comment]
That's right, IncFormLen and SkipFormLen should not be changed
